frontend/src/components/CANs/CANTable/CANTableRow.jsx

```diff
@@ -13,7 +12,7 @@ import { displayActivePeriod } from "./CANTableRow.helpers";
  * @param {Object} props
  * @param {number} props.activePeriod - Active Period in years
  * @param {number} props.canId - CAN ID
- * @param {number|"TBD"} props.fiscalYear - Fiscal Year
+ * @param {number} props.fiscalYear - Selected Fiscal Year
```

Comment:
So will that still be OK for cans without a set budget yet?  Pardon my ignorance

Reply:
the crux of the fix is using the selected fiscal year and not a CAN attribute. Think we fixed CANs without funding budgets in another PR.